src/test/resources/features/QuestionBank/QuestionBank.feature

```diff
@@ -0,0 +1,114 @@
+@QuestionBank @Regression
+Feature: Question Bank
+
+  Background:
+    Given user prepare question bank request
```

Comment:
I found it best to always call `And user hit logout endpoint` in each `Background`, since previous tests may alter the state of `authData` singleton